[2023-10-22T08:22:40.820+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_day18.fetch_data_from_psql scheduled__2023-10-21T04:00:00+00:00 [queued]>
[2023-10-22T08:22:40.833+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_day18.fetch_data_from_psql scheduled__2023-10-21T04:00:00+00:00 [queued]>
[2023-10-22T08:22:40.833+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-22T08:22:40.847+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): fetch_data_from_psql> on 2023-10-21 04:00:00+00:00
[2023-10-22T08:22:40.852+0000] {standard_task_runner.py:57} INFO - Started process 67 to run task
[2023-10-22T08:22:40.856+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_day18', 'fetch_data_from_psql', 'scheduled__2023-10-21T04:00:00+00:00', '--job-id', '128', '--raw', '--subdir', 'DAGS_FOLDER/ETL_day18.py', '--cfg-path', '/tmp/tmpbvalzkng']
[2023-10-22T08:22:40.860+0000] {standard_task_runner.py:85} INFO - Job 128: Subtask fetch_data_from_psql
[2023-10-22T08:22:40.913+0000] {task_command.py:415} INFO - Running <TaskInstance: etl_day18.fetch_data_from_psql scheduled__2023-10-21T04:00:00+00:00 [running]> on host 90cf752d844d
[2023-10-22T08:22:41.011+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kel11' AIRFLOW_CTX_DAG_ID='etl_day18' AIRFLOW_CTX_TASK_ID='fetch_data_from_psql' AIRFLOW_CTX_EXECUTION_DATE='2023-10-21T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-21T04:00:00+00:00'
[2023-10-22T08:22:41.024+0000] {base.py:73} INFO - Using connection ID 'postgresql_db' for task execution.
[2023-10-22T08:22:42.829+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/common/sql/hooks/sql.py:219: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  return psql.read_sql(sql, con=conn, params=parameters, **kwargs)

[2023-10-22T08:22:43.346+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-22T08:22:43.358+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=etl_day18, task_id=fetch_data_from_psql, execution_date=20231021T040000, start_date=20231022T082240, end_date=20231022T082243
[2023-10-22T08:22:43.381+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-22T08:22:43.411+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
